config/serverless-operator.yaml

```diff
@@ -12,6 +12,16 @@ config:
         version: "4.15"
       - onDemand: true
         version: "4.13"
+    release-1.35:
+      prowgen:
+        enabled: false
+      konflux:
+        enabled: false
```

Comment:
After the branch is actually created we'd flip this single flag to true. But we'd still keep prowgen.enabled == false

Reply:
SO is special cased for the flag https://github.com/openshift-knative/hack/blob/d565e2e9fbe0c39f0570ca737ebd3c0afc774683/pkg/prowgen/prowgen_konflux.go#L169-L199

so we don't even need this flag configuration nor flip it